packages/feedback-service/src/feedback-config/resolver.ts

```diff
@@ -30,14 +31,28 @@ const FeedbackConfigResolver = {
         .then((response: any) => response)
         .catch((error: Error) => error);
     },
+    getFeedbackConfigByProjectId(
+      root: any,
+      { projectId }: QueryListFeedbackConfigByAppIdArgs,
+      ctx: any,
+    ) {
+      return FeedbackConfig.findOne({ projectId })
```

Comment:
Kindly change to async/await. 

Also let it throw error apollo will catch it and format it

Reply:
@akhilmhdh we are returning the error. If the findOne fails it will be returned in the catch block

Reply:
For forward the error to apollo, the resolver needs to `throw` the error.

Reply:
![Screenshot 2023-01-30 at 2 09 03 PM](https://user-images.githubusercontent.com/12994292/215427884-602ac88e-c4dc-4898-9702-6c1d16593691.png)
@deshmukhmayur we are already getting that in apollo

Reply:
I introduced an error an it was caught in apollo

Reply:
it's exactly the same output with explicitly throwing an error explicitly or returning the error, unless we explicitly want to throw an error with some custom message which again is completely superfluous and unnecessary. It adds no value at all in this predicament. The error is already caught by the resolver. And still if you want me explicitly add the code I'll do it as this service is blocking other deployments but at least explain what value its adding.

Case in point:
This works fine (meaning the error is still caught by 
![Screenshot 2023-01-31 at 10 54 51 AM](https://user-images.githubusercontent.com/12994292/215673037-d764f31e-2dfb-4cb4-90e7-09f2c5bd71f4.png)

Additional code which isn't adding value
![Screenshot 2023-01-31 at 10 55 36 AM](https://user-images.githubusercontent.com/12994292/215673149-2d72261b-b4df-419c-a646-65108973fc93.png)

Reply:
Yes, but if you look closely at the `.then` and `.catch` methods, they're not doing anything. They're just returning the arguments back. So it's better to skip them altogether.
```suggestion
      return FeedbackConfig.findOne({ projectId }).exec();
```

Reply:
Yes absolutely, this makes much more sense. I'll update it right away.

Reply:
Updated